content/abstracts.rst

```diff
@@ -114,6 +114,10 @@ The PyCon UK panel will probably include:
  * Larry Hastings is one of the main CPython core contributors and the
    Release Manager for Python releases.
 
+ * Nicholas Tollervey is a Python trainer, former teacher, organiser
```

Comment:
Nicholas Tollervey is a freelance Python developer, classically trained musician, philosophy graduate, teacher, writer, organiser of the education track, founder of the London Python Code Dojo and PSF ambassador to the BBC micro:bit project. He enjoys spending his copious free time playing the tuba or hacking on distributed hash tables.